app/controllers/api/v1/games_controller.rb

```diff
@@ -15,8 +11,14 @@ def show
       end
 
       def create
-        require 'pry'; binding.pry
-        render json: Game.new()
+        player_1_key = request.headers['X-API-KEY']
+        player_2_key = User.find_by(email: params['opponent_email']).api_key
+        player_1 = Player.new(Board.new(4), player_1_key)
+        player_2 = Player.new(Board.new(4), player_2_key)
+
+        game = Game.create(player_1: player_1,
+                           player_2: player_2)
+        render json: game
       end
```

Comment:
So here I'm capturing the incoming variables we will need - player 1's api key and player 2's email(and api key). Then I'm using those to initialize new Player objects that contain (board, turns, and api keys).